snuba/manual_jobs/runner.py

```diff
@@ -128,9 +129,11 @@ def run_job(job_spec: JobSpec) -> JobStatus:
             current_job_status = _set_job_status(
                 job_spec.job_id, JobStatus.ASYNC_RUNNING_BACKGROUND
             )
+            _record_start_time(job_spec.job_id)
             job_to_run.execute(job_logger)
         else:
             current_job_status = _set_job_status(job_spec.job_id, JobStatus.RUNNING)
+            _record_start_time(job_spec.job_id)
             job_to_run.execute(job_logger)
             current_job_status = _set_job_status(job_spec.job_id, JobStatus.FINISHED)
             job_logger.info("[runner] job execution finished")
```

Comment:
We are doing pretty much the same things in the `if` and `else` blocks here. The difference is just the args passed. Can this be simplified?:
```
job_running = JobStatus.ASYNC_RUNNING_BACKGROUND if job_spec.is_async else JobStatus.RUNNING
try:
    current_job_status = _set_job_status(job_spec.job_id, job_running)
    _record_start_time(job_spec.job_id)
    job_to_run.execute(job_logger)
    if not job_spec.is_async:
        # Finish current job if no exception is raised and it is synchronous.
        current_job_status = _set_job_status(job_spec.job_id, JobStatus.FINISHED)
        job_logger.info("[runner] job execution finished")
except BaseException:
    ...
```

Reply:
This approach is much cleaner. Thank you Onkar!